tests/integration/zenodo_depositor_test.py

```diff
@@ -142,6 +142,11 @@ async def test_full_zenodo_flow(depositor, deposition_metadata, upload_key):
     assert latest_deposition.id_ == published_deposition.id_
 
     new_deposition = await depositor.get_new_version(published_deposition)
+    doubly_new_deposition = await depositor.get_new_version(new_deposition)
```

Comment:
who wants a new deposition when you can have _doubly_ new deposition

Reply:
> A new deposition isn't cool, you know what's cool? A doubly new deposition.